File: paddock/data/fixtures/okf-behavior-php/source/service.php
<?php
function dispatch(array $items, array &$sent, int $limit): array
{
    if ($limit < 0) {
        throw new InvalidArgumentException("limit must be nonnegative");
    }
    if (count($items) === 0) {
        return ["status" => "empty", "count" => 0];
    }
    $selected = array_slice($items, 0, min($limit, count($items)));
    array_push($sent, ...$selected);
    return ["status" => "sent", "count" => count($selected)];
}

File: paddock/data/fixtures/okf-behavior/source/service.py
"""A bounded dispatch operation with observable errors, defaults and effects."""
from __future__ import annotations


def dispatch(items: list[str], sent: list[str], limit: int = 2) -> dict[str, object]:
    if limit < 0:
        raise ValueError("limit must be nonnegative")
    if not items:
        return {"status": "empty", "count": 0}
    selected = items[:limit]
    sent.extend(selected)
    return {"status": "sent", "count": len(selected)}
